### ISSUE TITLE
Minor typo in the NIXPY API documentation

### ISSUE THREAD

**Issue by msuzen** [G-Node/nixpy#241]
In the API documentation [here](http://g-node.github.io/nixpy/overview.html#design-principles)
library call is
 
`import nix` 

probably should be

 `import nixio`

**Comment by jgrewe** [comment]
that's true, apparently we missed a few spots. Actually the same applies to most of the tutorial code.
To make it easy, the import statement an be changed to:

````
import nixio as nix
````

**Comment by achilleas-k** [comment]
Documentation generally needs some love, both the usage docs and the API.

**Comment by s0nskar** [comment]
Isn't it weird in [overview.html](https://github.com/G-Node/nixpy/blob/master/docs/source/overview.rst#L24) Line 24 it is written `import nixio as nix` but on [API documentation](http://g-node.github.io/nixpy/overview.html) it is showing `import nix`?

**Comment by jgrewe** [comment]
This is for historical reasons :) 
In former times, when the documentation was written the package was named nix...

**Comment by JuliaSprenger** [comment]
I agree the docu should be updated.  Also I think it's misleading to put examples of failure in the basic introduction like here (http://g-node.github.io/nixpy/tutorial.html#working-with-files). This rather belongs to troubleshooting in my opinion.
